lib/set/setJSONGraphs.js

```diff
@@ -118,18 +118,23 @@ function setJSONGraphPathSet(
 }
 /* eslint-enable */
 
+var _result = new Array(4);
```

Comment:
This one is admittedly a little ugly. I noticed that the return values of these local functions that return arrays of 4 elements never live long enough to overlap with another call. This means we can just reuse a single array for their return value instead of creating a new one each time.